qcost

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
```

Comment:
check the shbang of uqstat, that uses conda's python always

Reply:
Tah